<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualWorldClass</class>
 <widget class="QMainWindow" name="VirtualWorldClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VirtualWorld</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="dragMode">
          <enum>QGraphicsView::NoDrag</enum>
         </property>
         <property name="viewportUpdateMode">
          <enum>QGraphicsView::SmartViewportUpdate</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>Objects</string>
         </property>
         <widget class="QRadioButton" name="radioButton_Circle">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>89</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Circle</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="radioButton_Square">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>89</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Square</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="radioButton_Rectangle">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>89</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Rectangle</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
         <widget class="QPushButton" name="Object_GroupButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Object_AddButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Object_RemoveButton">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>90</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
